.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+## Description
+<!-- A few sentences describing the overall goals of the pull request' s commits. -->
+
+**Please check if the PR fulfills these requirements**
+- [ ] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
```

Comment:
This project should not be tied to MapStore2, also for the example I would use a more generic template (I would look if anyone created some generic templates to inherit)

Reply:
sounds good
what about this one?
https://github.com/stevemao/github-issue-templates/blob/master/questions-answers/PULL_REQUEST_TEMPLATE.md

an alternative would be to remove the link to the guidelines and leave the rest